<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameMainmenu</class>
 <widget class="QWidget" name="GameMainmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>1000</width>
     <height>800</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>800</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>game_resources/game_ui_image2(1).jpg</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>212</width>
     <height>344</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="Settings_pushbutton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/game_resources/button_background1(1).png);</string>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Exit_pushbutton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/game_resources/button_background1(1).png);</string>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Helper_pushbutton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/game_resources/button_background1(1).png);</string>
      </property>
      <property name="text">
       <string>Helper</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="Start_pushbutton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/game_resources/button_background1(1).png);</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
      <property name="iconSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
